docs/examples/general/data_loading/dataloading_webdataset.ipynb

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
<p>I think we need a webdataset build from https://github.com/NVIDIA/DALI_extra/tree/main/db/single/jpeg. We did that for RecordIO, TFRecord, Caffe &amp; Caffe2 LMDB. So having a corresponding WebDataset would be perfect.</p><p>I don't know how much it is about this PR but test_RN50_data_pipeline.py can be extended as well (at least the case with small_test_data as we don't have a full ImageNet in that format in our CI).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>This log looks strange (two entries with 0 stage):</p><pre class="ql-syntax" spellcheck="false">time: 0.00 count: 0 stage: collect 
time: 0.01 count: 0 stage: collect 
time: 0.01 count: 141 stage: done 
</pre>

Reply:
<p>alright</p>